tests/test_tools.py

```diff
@@ -0,0 +1,27 @@
+#!/usr/bin/env pytest -vs
+"""Tests for Validation functions."""
+
+# Third-Party Libraries
+import pytest
+
+# cisagov Libraries
+from tools.gophish_complete import get_campaign_id
+
+
+class TestComplete:
+    """Test gophis-complete script."""
```

Comment:
Typo: `gophis-complete` -> `gophish-complete`

Reply:
Correction made in commit b9dbc4f.